cocos/renderer/CCTextureAtlas.h

```diff
@@ -165,38 +164,37 @@ class CC_DLL TextureAtlas : public Ref
      This method doesn't enlarge the array if newIndex + quads to be moved > capacity
      @since 1.1
     */
-    void moveQuadsFromIndex(ssize_t index, ssize_t newIndex);
+    void moveQuadsFromIndex(size_t index, size_t newIndex);
 
     /**
      Ensures that after a realloc quads are still empty
      Used internally by ParticleBatchNode
      @since 1.1
     */
-    void fillWithEmptyQuadsFromIndex(ssize_t index, ssize_t amount);
+    void fillWithEmptyQuadsFromIndex(size_t index, size_t amount);
 
     /** draws n quads
     * n can't be greater than the capacity of the Atlas
     */
-    void drawNumberOfQuads(ssize_t n);
+    void drawNumberOfQuads(size_t n);
 
     /** draws n quads from an index (offset).
     n + start can't be greater than the capacity of the atlas
 
     @since v1.0
     */
-    void drawNumberOfQuads(ssize_t numberOfQuads, ssize_t start);
+    void drawNumberOfQuads(size_t numberOfQuads, size_t start);
```

Comment:
it should be `ssize_t`... it means `signed`... by being signed it is easier to detect possible bugs. 
It is defined in the Cocos2d-x C++ guidelines.